google-http-client/src/main/java/com/google/api/client/http/HttpResponse.java

```diff
@@ -351,9 +351,9 @@ public InputStream getContent() throws IOException {
         try {
           // gzip encoding (wrap content with GZipInputStream)
           if (!returnRawInputStream && this.contentEncoding != null) {
-            String oontentencoding = this.contentEncoding.trim().toLowerCase(Locale.ENGLISH);
-            if (CONTENT_ENCODING_GZIP.equals(oontentencoding)
-                || CONTENT_ENCODING_XGZIP.equals(oontentencoding)) {
+            String contentencoding = this.contentEncoding.trim().toLowerCase(Locale.ENGLISH);
```

Comment:
contentEncoding

Reply:
also please triple check that the field isn't read somewhere below without `this`

Reply:
It is not. Will rename to `contentEncoding`